index.js

```diff
@@ -741,7 +741,7 @@ Writer.prototype.append = function (entry, cb) {
   mapped.clock = this._mapList(entry.clock, this._encodeMap, 0)
   mapped.inflate = this._feeds
   mapped.trie = trie.encode(entry.trie, this._encodeMap)
-  if (entry.value !== null) mapped.value = this._db._valueEncoding.encode(entry.value)
+  if (!isNullish(entry.value)) mapped.value = this._db._valueEncoding.encode(entry.value)
```

Comment:
Doesn't make sense for any valueEncodings to do special things with nulls right?

Reply:
No that's what I'm thinking